docs/index.md

```diff
@@ -25,7 +25,7 @@ From within an empty folder, add a minimal HTML file called `index.html`:
     ContainerJS status: <span id="status">initialising</span>
   </div>
 
-  <script src="https://unpkg.com/containerjs-api-bundle@0.0.2/build/containerjs-bundle.js"></script>
+  <script src="/scripts/containerjs-api.js"></script>
```

Comment:
Yeah me too, but it would be out of date until we publish to npm again. Unless you think we should do that immediately after this PR?